### PR TITLE
Feat filter and portraits scaffold

### PR DESCRIPTION
**Pull request by mixcas**
- scaffold the portraits grid using isotope
- scaffold filter selects
- add filters functionality

### CODE REVIEW

File: front-page.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+
+  <?php get_template_part('partials/portraits'); ?>
+
+  <?php get_template_part('partials/pagination'); ?>
```
Comment (varffs) [inline front-page.php]: do we want this if were have this portraits section?

File: src/js/main.js [hunk 2]
```diff
@@ -84,4 +89,83 @@ Site.Paypal = {
...
+    // Set grid with isotope
+    _this.$grid = $('#portraits-grid').isotope({
+      // options
+      itemSelector: '.grid-item',
+      layoutMode: 'fitRows'
+    });
+
+    // Workaround to make it compatible with lazysizes
+    _this.$grid[0].addEventListener('load', (function(){
+      var runs;
```
Comment (varffs) [inline src/js/main.js:122]: could you comment this function please

**Review by varffs — COMMENTED** [review]
Seems good and works. Only potential bug is that is doesnt support multiple selections for taxonomies. So if a tax is like Subject or whatever and the Portrait has 2 subjects selected then it is only ever possible to filter via the first one. That seems like a problem to me. 

I think other details can be done in detailing branches. Like for example we need a no results state for when filtered to nothing. And maybe a reset button for the filters. etc

File: src/js/main.js (hunk 2 above)
Comment (mixcas) [reply src/js/main.js:122]: i dont know how it works, i cant find the source either :(
but it works

**Comment by mixcas** [comment]
@varffs I added support for multiple taxonomies and added comments. also removes pagination from frontpage

File: src/js/main.js (hunk 2 above)
Comment (varffs) [reply src/js/main.js:122]: we should try to just comment the link to the source when we use snippets like this imo

Comment (mixcas) [reply src/js/main.js:122]: i know but i couldnt find it :(

**Pull request merged**